### PR TITLE
Added HttpClient and ClientService

### PR DESCRIPTION
**Pull request by vicire**

### CODE REVIEW

File: src/main/java/bankservice/demo/service/TransactionServiceImpl.java [hunk 1]
```diff
@@ -50,8 +54,13 @@ public void transfer(Account fromAccount, Account toAccount, BigDecimal amount)
         transactionIn.setAmount(amount);
```
Comment (KseniiaMakarova) [inline src/main/java/bankservice/demo/service/TransactionServiceImpl.java]: let's set the updated amount here as well

File: src/main/java/bankservice/demo/service/util/ClientService.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ClientService {
+    @Value(value = "$(service.url)")
+    private String value;
+    private String url;
```
Comment (KseniiaMakarova) [inline src/main/java/bankservice/demo/service/util/ClientService.java]: do we use both fields?

**Review by KseniiaMakarova — APPROVED** [review]

File: src/main/java/bankservice/demo/service/util/ClientService.java (hunk 2 above)
Comment (vicire) [reply src/main/java/bankservice/demo/service/util/ClientService.java]: Fixed

File: src/main/java/bankservice/demo/service/TransactionServiceImpl.java (hunk 1 above)
Comment (vicire) [reply src/main/java/bankservice/demo/service/TransactionServiceImpl.java]: Fixed

**Pull request merged**